### PR TITLE
Add commit-weather

### PR DESCRIPTION
**Pull request by larowlan**
container for core committers to track current freeze state

What's the weather like for committing? `/weather status`
Flag that we're in freeze `/weather freeze Waiting for 8.7.2 release`
or good to commit `/thaw`
or find out when the freeze ended `/weather extended-status`
Get help `/weather help`

### CODE REVIEW

File: commit-weather/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM node:10-slim
```
Comment (kimpepper) [inline commit-weather/Dockerfile]: Why not use the pnx node image?

File: commit-weather/Makefile [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+	docker build -t $(IMAGE):$(VERSION) .
+
+lint:
+	hadolint Dockerfile
+
+# Build and release
+release: build
+	docker push $(IMAGE):$(VERSION)
+
+.PHONY: build lint release
```
Comment (kimpepper) [inline commit-weather/Makefile]: `.PHONY: *` works here

File: commit-weather/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+WORKDIR /usr/src/app
+
+COPY package*.json ./
+
+RUN npm install --only-production
+
+EXPOSE 3000
```
Comment (kimpepper) [inline commit-weather/Dockerfile]: Can we just use port 80?

Comment (larowlan) [reply commit-weather/Dockerfile]: it runs on 3000 by default

File: commit-weather/Makefile (hunk 2 above)
Comment (larowlan) [reply commit-weather/Makefile]: borrowed from the existing ones

File: commit-weather/Dockerfile (hunk 1 above)
Comment (larowlan) [reply commit-weather/Dockerfile]: we don't need make and all the other stuff it adds

**Comment by larowlan** [comment]
will create a new repo

**Pull request closed without merging**